api/proto/config/devmodel.proto

```diff
@@ -141,12 +138,21 @@ message PhysicalIO {
       // EntireGroup can be assigned to application or nothing at all
       string  assigngrp         = 5;
 
-      // XXX Note that this is the static information from the model.
-      // Current configuration is in systemAdapter
+      // usage - indicates the role of adapter ( mgmt / blocked / app-direct
+      //    etc. )
       PhyIoMemberUsage usage    = 6;
+
+      // usagePolicy - Policy Object used to further refine the usage.
+      // For example, specify if this should be only used as fallback?
+      //    Or used as the primary uplink? Allow App traffic? restrict
+      //    app traffic?? etc..
       PhyIOUsagePolicy usagePolicy    = 7;
 
       // physical and logical attributes
       //    For example in WWAN to which firmware version to laod etc
       map <string, string> cbattr = 8;
+
+      // systemAdapters - List of system Adapters using this IoAdapter.
+      // key: SystemAdapter.Name
+      map <string, SystemAdapter> systemAdapters = 11;
```

Comment:
Why is this inside each physicalIO as opposed to being a global map?

Reply:
Removed this.. made the changes as per our discussion.